.circleci/config.yml

```diff
@@ -146,3 +145,50 @@ jobs:
                     --attach build/distributions/git-machete-intellij-plugin-*.zip \
                     --message "$tag"$'\n\n'"$change_notes" \
                     $tag
+  checkForNewEapBuildNumber:
+    docker:
+      - image: 'gitmachete/intellij-plugin-ci:6.1.0'
+    working_directory: ~/git-machete-intellij-plugin
+    resource_class: large
+    steps:
+      - checkout
+      - run:
+          name: Reconfigure origin remote to https
+          command: git remote set-url origin https://${GITHUB_TOKEN}@github.com/VirtusLab/git-machete-intellij-plugin.git
+      - run:
+          name: Configure email and name
+          command: |
+            git config user.email "cron_job@example.com"
+            git config user.name "cron_job"
+      - run:
+          # language=sh
+          command: |
+            ./gradlew updateEapBuildNumber
+            if [[ `git status --porcelain` ]]; then
+              newEapBuildNumber=$(git diff intellijVersions.properties | grep -o '+eapOfLatestSupportedMajor.*' | cut -d= -f2)
+              branch=updateEap/$newEapBuildNumber
+              git checkout -b "$branch"
+              git add .
+              git commit -m "New eap build number $newEapBuildNumber"
```

Comment:
Nit: pls use imperative form in commit messages, see https://cbea.ms/git-commit/#imperative